addons/account_check_printing/models/account_journal.py

```diff
@@ -34,6 +34,16 @@ def _default_outbound_payment_methods(self):
         help="Sequence number of the next printed check.",
     )
 
+    bank_check_printing_layout = fields.Selection(
+        selection='_get_check_printing_layouts',
+        string='Check Layout'
+    )
```

Comment:
Maybe we could put "account_check_printing_layout" as a related field instead of creating a new one completely and if you want to have the string called Check Layout just put a string parameter in the account_journal_views

Reply:
It is important to note that using `account_check_printing_layout` as a related field in the selection field within the bank settings journal will prevent us from having distinct values for the layout in both the general settings and bank journal settings.  
Any changes made to the layout in the bank settings will automatically update the layout in the general settings. Therefore, it is not possible to have separate layouts for both settings.

Reply:
Ok i see why you made a new field then, to do some nitpicking here add the coma at the end of the string so that if someone want to edit the field later he don't break the history. Also a nice thing to know when to put single or double quotes is usually if the parameter is something that the client will see then put double quotes. So for the string i would put double quotes and keep single quotes for the selection

Reply:
Great, I'll update the PR. Thank you for reviewing it.